website/events/api/v2/admin/views.py

```diff
@@ -72,9 +72,17 @@ class EventRegistrationAdminListView(AdminListAPIView, AdminCreateAPIView):
     required_scopes = ["events:admin"]
     filter_backends = (
         framework_filters.OrderingFilter,
+        framework_filters.SearchFilter,
         filters.EventRegistrationCancelledFilter,
     )
     ordering_fields = ("queue_position", "date", "date_cancelled")
+    search_fields = (
+        "member__profile__nickname",
```

Comment:
We don't use nicknames for users in the admin anywhere, don't think we need this. Really weird if you see all the Jens show up in the admin but don't know why.